www-apps/cgp/cgp-1.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit webapp
+
+MY_PN="${PN^^}"
+
+DESCRIPTION="A graphical web-based front-end for visualizing RRD collected by collectd"
+HOMEPAGE="https://pommi.nethuis.nl/category/cgp/"
+SRC_URI="https://github.com/pommi/${MY_PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+KEYWORDS="~amd64 ~x86"
+IUSE="+json"
+
+RDEPEND="net-analyzer/rrdtool[graph]
+	json? ( dev-lang/php[json] )
```

Comment:
why do you want to enforce json flag here? It is enforced by the php ebuilds itself as the very few basic set.

Reply:
Well, I removed the use flag. But even in that case, I would always depend on it. What if an user does USE="-*"?

Reply:
then you might want something like

`IUSE="json"`
...
`RDEPEND="json? ( dev-lang/php[json?] )"`

in cause you enable json for this package it will also enable json for php